packages/types/src/inventory/common.ts

```diff
@@ -202,6 +202,20 @@ export type InventoryLevelDTO = {
   deleted_at: string | Date | null
 }
 
+/**
+ * @interface
+ * 
+ * The filters to apply on retrieved reservation items.
+ * 
+ * @prop id - The IDs to filter reservation items by.
+ * @prop type - The types to filter reservation items by.
+ * @prop line_item_id - Filter reservation items by the ID of their associated line item.
+ * @prop inventory_item_id - Filter reservation items by the ID of their associated inventory item.
+ * @prop location_id - Filter reservation items by the ID of their associated location.
+ * @prop description - Description filters to apply on the reservation items' `description` attribute.
+ * @prop created_by - The "created by" values to filter reservation items by.
+ * @prop quantity - Filters to apply on the reservation items' `quantity` attribute.
+ */
 export type FilterableReservationItemProps = {
   id?: string | string[]
   type?: string | string[]
```

Comment:
We don't have reservation types anymore
```suggestion
 * @prop id - The IDs to filter reservation items by.
 * @prop line_item_id - Filter reservation items by the ID of their associated line item.
 * @prop inventory_item_id - Filter reservation items by the ID of their associated inventory item.
 * @prop location_id - Filter reservation items by the ID of their associated location.
 * @prop description - Description filters to apply on the reservation items' `description` attribute.
 * @prop created_by - The "created by" values to filter reservation items by.
 * @prop quantity - Filters to apply on the reservation items' `quantity` attribute.
 */
export type FilterableReservationItemProps = {
  id?: string | string[]
```